docs/scos/dev/glue-api-guides/202311.0/decoupled-glue-api.md

```diff
@@ -1,36 +1,36 @@
 ---
 title: Decoupled Glue API
 description: Learn about the process of handling API requests through GlueStorefront and GlueBackoffice layers.
-last_updated: Jul 11, 2023
+last_updated: November 29, 2023
 template: glue-api-storefront-guide-template
 ---
 
-The Spryker Decoupled Glue API is a set of a few API applications like *Glue Storefront API (SAPI)* and *Glue Backend API (BAPI)* of the Spryker Commerce OS. Those applications are built to be used as a contract for accessing Storefront or Backoffice functionality through API. Those applications know how to read and interpret API resources and leverage feature modules that expose existing Spryker functionality.
+The Spryker Decoupled Glue API is a set of a few API applications like *Glue Storefront API* and *Glue Backend API* of the Spryker Commerce OS. Those applications are built to be used as a contract for accessing Storefront or Backoffice functionality through API. Those applications know how to read and interpret API resources and leverage feature modules that expose existing Spryker functionality.
```

Comment:
Please call it Spryker Cloud Commerce OS and add a reference to the application description